src/Examine.Lucene/Search/LuceneSearchQueryBase.cs

```diff
@@ -328,10 +328,10 @@ protected virtual Query GetFieldInternalQuery(string fieldName, IExamineValue fi
                     if (useQueryParser)
                     {
                         queryToAdd = _queryParser.GetFieldQueryInternal(fieldName, fieldValue.Value);
-                        if (queryToAdd != null)
-                        { 
+                        //if (queryToAdd != null) // TODO: Does this ever happen? Then the method should be made with a nullable return
```

Comment:
I think it can. This is the underlying code that is called from GetFieldQueryInternal:

![image](https://github.com/Shazwazza/Examine/assets/1742685/8996a0d1-7304-4b61-9c1d-3297ff2a451b)

So I guess this could return null too then.

Reply:
Super I marked it accordingly